backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -930,6 +963,11 @@ public Set<Observation> convertToAOEObservations(
       observations.addAll(convertToAOEResidenceObservation(residesInCongregateSetting, null));
     }
 
+    List<String> sexualPartners = surveyData.getGenderOfSexualPartners();
```

Comment:
It might be worth trying to turn this list into a set here because as written, any duplicates for the value of sexual partners here (ie in a CSV upload) would generate duplicate entries in the end bundle.

Reply:
I initially tried to have it as a set but inteliJ got mad because the gender of sexual partners attribute is defined as a list on the surveyData object. Should I go ahead and change how we define it on the object?

Reply:
Don't think we need to change it on `surveyData`. Think you can just convert it to a set in the `convertToAOEGenderOfSexualPartnersObservation` [using the HashSet constructor](https://stackoverflow.com/questions/1429860/easiest-way-to-convert-a-list-to-a-set-in-java) and then iterate over the set instead.